content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+# Updating Bottlerocket
+
+This document covers the different ways to update Bottlerocket (for [ECS](#ecs), [Kubernetes](#kubernetes), and [SSM](#ssm)), as well as how to [lock your nodes to a specific release](#locking-to-a-specific-release).
+
+## Introduction
+
+Bottlerocket is designed to be updated in an [image-based fashion](https://github.com/bottlerocket-os/bottlerocket#updates).
+This means that updates are applied by replacing the entire image on disk, rather than applying a series of individual package updates.
+This is a departure from the traditional package-based Linux update model such as `apt` or `yum`, which are what you would find in Ubuntu or Amazon Linux.
+
+## Ways To Update
```

Comment:
Describe logging in to the control container and doing this with the apiclient first, then explain that we have these orchestrated systems for doing it automatically.